awscrt/awsiot_mqtt_connection_builder.py

```diff
@@ -182,7 +182,7 @@ def _builder(
         clean_session=kwargs.get('clean_session', False),
         reconnect_min_timeout_secs=kwargs.get('reconnect_min_timeout_secs', 5),
         reconnect_max_timeout_secs=kwargs.get('reconnect_max_timeout_secs', 60),
-        keep_alive_secs=kwargs.get('keep_alive_secs', 3600),
+        keep_alive_secs=kwargs.get('keep_alive_secs', 1200),
```

Comment:
Docs say this value must be higher than ping_timeout_ms, but that is not the case in the defaults.

Reply:
One value is in seconds, the other in milliseconds. Changed the wording of the comments slightly to try and make this less confusing
~~This value must be higher than ping_timeout_ms.~~
This duration must be longer than ping_timeout_ms.